app/src/main/java/com/lagradost/cloudstream3/ui/settings/SettingsUI.kt

```diff
@@ -20,6 +24,9 @@ import com.lagradost.cloudstream3.utils.SingleSelectionHelper.showMultiDialog
 import com.lagradost.cloudstream3.utils.UIHelper.hideKeyboard
 
 class SettingsUI : PreferenceFragmentCompat() {
+    companion object {
+        fun getConfirmExitDialogDefault() = isLayout(TV or EMULATOR)
```

Comment:
This is useless impo

Reply:
I would agree but since multiple people contribute this fun makes it difficult to introduce bugs, say when we decide to change the default value in a certain layou.